### ISSUE TITLE
Add. 12,137, foll. 2–177 | https://bl.syriac.uk/ms/1161 Link to Wright missing

### ISSUE THREAD

**Issue by bogdg** [srophe/britishLibrary-data#1835]
I'm not sure if the issue is at this line but there's no link to the catalogue for these entries

<img width="329" alt="image" src="https://github.com/user-attachments/assets/7a62f9ac-acdb-4b33-9971-777729789c17">

https://github.com/srophe/britishLibrary-data/blob/7ed9d834c099761265cdcb643468a8d01e06be53/data/tei/9.xml#L294

**Comment by davidamichelson** [comment]
Good catch, yes that is the source of the problem. @wlpotter Can you figure out what should go there please?

**Comment by wlpotter** [comment]
This is odd. The bibl and url for archive.org all appear to be correct, see [these lines](https://github.com/srophe/britishLibrary-data/blob/main/data/tei/9.xml#L404-L413) for example.

Let's leave this issue open for now and I can look at it more closely. It could maybe be something to do with the fact that this record has descriptions in the containing manuscript and in the parts, though I don't know why that would affect the link to the PDF...
